elements/Shape.js

```diff
@@ -10,6 +11,17 @@ class Shape extends Component {
         });
         this.state = this.touchableGetInitialState();
     }
+
+    setNativeProps = (args) => {
+        var extracted = extractProps(args, this)
```

Comment:
This however doesn't work for transform matrices. Ex. I want to specify a translation, if I specify in {transform: {translateX: 12}} the for loop afterwards will filter out the matrix extracted.